okapi-core/src/main/java/org/folio/okapi/managers/DeploymentManager.java

```diff
@@ -62,11 +73,17 @@ public DeploymentManager(Vertx vertx, DiscoveryManager dm, EnvManager em,
     this.nodeName = nodeName;
     this.eventBus = vertx.eventBus();
     this.config = config;
-    int portStart = Integer.parseInt(Config.getSysConf("port_start", Integer.toString(listenPort + 1), config));
-    int portEnd = Integer.parseInt(Config.getSysConf("port_end", Integer.toString(portStart + 10), config));
+    int portStart = Integer.parseInt(Config.getSysConf(
+      "port_start", Integer.toString(listenPort + 1), config));
+    int portEnd = Integer.parseInt(Config.getSysConf(
+      "port_end", Integer.toString(portStart + 10), config));
     this.ports = new Ports(portStart, portEnd);
   }
 
+  /**
+   * Unitialize deployment manager.
```

Comment:
This is contrary to the method name `init` and I don't see it in the code, can you be more verbose?

Reply:
A typo. Good catch!